<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LanguageDialog</class>
 <widget class="QDialog" name="LanguageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 51, 51);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>545</width>
     <height>140</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border: 1px solid white;
font: 20px 'Roboto';</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(190, 190, 190);
border: 0px;
font: 16px 'Roboto';</string>
    </property>
    <property name="text">
     <string>English</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(190, 190, 190);
border: 0px;
font: 16px 'Roboto';</string>
    </property>
    <property name="text">
     <string>Português</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(190, 190, 190);
border: 0px;
font: 16px 'Roboto';</string>
    </property>
    <property name="text">
     <string>ไทย</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(190, 190, 190);
border: 0px;
font: 16px 'Roboto';</string>
    </property>
    <property name="text">
     <string>Indonesia</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px;
	image: url(:/images/images/USAPassiveImage.png);
}

QPushButton:hover
{
	border: 0px;
	image: url(:/images/images/USAActiveImage.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px;
	image: url(:/images/images/brasilPassiveImage.png);
}

QPushButton:hover
{
	border: 0px;
	image: url(:/images/images/brasilActiveImage.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px;
	image: url(:/images/images/thailandPassiveImage.png);
}

QPushButton:hover
{
	border: 0px;
	image: url(:/images/images/thailandActiveImage.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px;
	image: url(:/images/images/indonesiaPassiveImage.png);
}

QPushButton:hover
{
	border: 0px;
	image: url(:/images/images/indonesiaActiveImage.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px;
	image: url(:/images/images/turkeyPassiveImage.png);
}

QPushButton:hover
{
	border: 0px;
	image: url(:/images/images/turkeyActiveImage.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(190, 190, 190);
border: 0px;
font: 16px 'Roboto';</string>
    </property>
    <property name="text">
     <string>Türkçe</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
